### PR TITLE
Dutch Localization

### PR DESCRIPTION
**Pull request by HeresHavi**

### CODE REVIEW

File: src/main/resources/assets/toolbelt/lang/nl_nl.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "item.toolbelt.belt": "Gereedschapsgordel",
+  "item.toolbelt.pouch": "Heuptasje",
+  "key.toolbelt.open": "Wissel tool",
```
Comment (martmists) [inline src/main/resources/assets/toolbelt/lang/nl_nl.json:4]: Wissel gereedschap

File: src/main/resources/assets/toolbelt/lang/nl_nl.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "key.toolbelt.category": "Gereedschapsgordel",
+  "key.toolbelt.slot": "Open inventaris van riem",
+  "key.toolbelt.cycle.left": "Cyclustool naar links",
+  "key.toolbelt.cycle.right": "Cyclustool naar rechts",
+  "text.toolbelt.empty": "LEEG",
+  "text.toolbelt.swap": "RUIL",
+  "text.toolbelt.insert": "INVOEGEN",
+  "text.toolbelt.extract": "EXTRACT",
+  "text.toolbelt.tooltip": "Upgrades: %s (%s slots)",
```
Comment (martmists) [inline src/main/resources/assets/toolbelt/lang/nl_nl.json:13]: `Verbeteringen: %s ...`

**Comment by gigaherz** [comment]
Huh I somehow missed this PR was open and waiting, my bad!

**Comment by gigaherz** [comment]
I'll merge the PR then apply your suggestions @martmists-gh 
Sorry that I forgot to merge last year.

**Pull request merged**